code-challenges.test.js

```diff
@@ -38,32 +58,135 @@ const num3 = -7
 
 // b) Create the function that makes the test pass.
 
+// Pseudocode: 
+// Declare a function called evenDivisiblebyThree.
+// Set parameters referenced as num1, num2, num3. 
+// Use conditional to evalute which inputs are are divisible by three.
+// Return the specific numbers that are divisible and which ones aren't divisible by three. 
+
+// const evenDivisiblebyThree = (num1, num2, num3) => {
+//     if (num1 % 3==0) {
+//         return "15 is divisible by three"
+//     }   else if (num2 % 3==0){
+//         return "0 is divisible by three"
+//     }   else if (num3 % 3!==0){
+//         return "-7 is not divisible by three"
+//     }
+// }
+
+ //* RAN TEST (RESULTS BELOW): *//
+
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
 
+//   ReferenceError: Cannot access 'evenDivisiblebyThree' before initialization
+
+// Note to Instructor: I looked up the error and according to my research this error occurs becuase the the function set up is trying to access the variable before it's initialised so I tried restructuring my code but was getting syntax errors so I'ma bit lost because it looks correct to me. Anyway, this is why I have commented out my #1 - code lines, so I can still run tests for the other function problems.
```

Comment:
Hi Steven.  Your pseudocode and test look great!  You are getting this error because in your describe method in the test, the function name is not in quotes.  Also,  be sure your code is matching the question- this function takes should take in one number and return if that specific number(argument) is divisible by three.  To make it dynamic, you don't want to hardcode the return. (think string interpolation).  Hope this help!

Reply:
That makes a lot of senses, thank you for the input...will definitely look for such things going forward.